src/js/pages/jobs/JobRunHistoryTable.js

```diff
@@ -0,0 +1,223 @@
+      let dateRunStarted = activeRun.getDateCreated();
+      let dateRunFinished = longestRunningTask.getDateCompleted();
+
+      if (dateRunStarted != null) {
+        dateRunStarted = DateUtil.msToRelativeTime(dateRunStarted);
+      }
+
+      if (dateRunFinished != null) {
+        dateRunFinished = DateUtil.msToRelativeTime(dateRunFinished);
+      }
+
+      let children = activeRun.getTasks().getItems().map(function (jobTask) {
+        let dateTaskStarted = jobTask.getDateStarted();
+        let dateTaskFinished = jobTask.getDateCompleted();
+
+        if (dateTaskStarted != null) {
+          dateTaskStarted = DateUtil.msToRelativeTime(dateTaskStarted);
+        }
+
+        if (dateTaskFinished != null) {
+          dateTaskFinished = DateUtil.msToRelativeTime(dateTaskFinished);
+        }
+
+        return {
+          taskID: jobTask.getTaskID(),
+          status: jobTask.getStatus(),
+          startedAt: dateTaskStarted,
+          finishedAt: dateTaskFinished
+        };
+      });
+
+      return {
+        finishedAt: dateRunFinished,
+        id: runIndex,
+        jobID: activeRun.getJobID(),
+        startedAt: dateRunStarted,
+        status: activeRun.getStatus(),
+        children
+      };
+    });
+  }
+
+  renderJobIDColumn(prop, row, rowOptions = {}) {
+    if (!rowOptions.isParent) {
+      return (
+        <div className="job-run-history-task-id text-overflow">
+          {row.taskID}
+        </div>
+      );
+    }
+
+    let classes = classNames('job-run-history-job-id', {
+      'is-expanded': rowOptions.isExpanded
+    });
+    let clickHandler = null;
+
+    if (rowOptions.hasChildren) {
+      clickHandler = this.handleExpansionClick.bind(this, row);
+    }
+
+    return (
+      <div className={classes} onClick={clickHandler}>
+        {row[prop]}
+      </div>
+    );
+  }
+
+  renderStatusColumn(prop, row, rowOptions = {}) {
+    if (rowOptions.isParent) {
+      let status = JobStates[row[prop]];
+      let statusClasses = classNames({
+        'text-success': status.stateTypes.includes('success')
+          && status.stateTypes.includes('failure'),
+        'text-danger': status.stateTypes.includes('failure'),
+        'text-neutral': status.stateTypes.includes('active')
+      });
+
+      return (
+        <span className={statusClasses}>
+          {status.displayName}
+        </span>
+      );
+    }
+
+    let status = TaskStates[row[prop]];
+    let statusClasses = classNames({
+      'text-success': status.stateTypes.includes('success')
+        && status.stateTypes.includes('failure'),
+      'text-danger': status.stateTypes.includes('failure')
+    });
+
+    return (
+      <span className={statusClasses}>
+        {status.displayName}
+      </span>
+    );
+  }
+
+  render() {
+    let job = ChronosStore.getJob(this.props.jobID);
+    let activeRunCount = job.getActiveRuns().getItems().length;
+
+    let filterHeadlineLabel = StringUtil.pluralize('Run', activeRunCount);
```

Comment:
`FilterHeadline` now takes care of capitalising itself (:

![Runss](http://cl.ly/45263y1p3213/Image%202016-06-15%20at%2010.27.24%20AM.png)